backend/AAiT-backend-group-12/domain/domain.go

```diff
@@ -0,0 +1,40 @@
+package domain
+
+import (
+	"time"
+)
+
+type Blog struct {
+	ID         string
+	Title      string
+	Content    string
+	UserID     string
```

Comment:
I thought we were associating the user's user name here instead of the primitive object id. We will here Nathan's thought on this.